README.md

```diff
@@ -1,25 +1,38 @@
+[![SublimeHQ Discord](https://img.shields.io/discord/280102180189634562?label=SublimeHQ%20Discord&logo=discord&style=flat-square)](https://discord.gg/D43Pecu)
+
 # Sublime Packages
 
-> **These packages are developed against the latest [Sublime Text 3 Dev Build](http://sublimetext.com/3dev). Bugs may exist on older builds, and the format used is not compatible with builds older than 3092.**
+> **These packages are developed against the latest [Sublime Text Dev Build](http://sublimetext.com/).**
+> **Bugs may exist on older builds, and the format used is not compatible with builds older than 4075.**
```

Comment:
FWIW, some syntaxes (Java Rewrite) may even fail on builds before 4094 or even cause older versions to crash or freeze.

Reply:
@deathaxe The changelog doesn't mention any syntax related changes for ST4094. Are you sure about this?

Reply:
Since #2655 bumped the test binary from 4090 to 4094 and fixed tests failing in #2654, it must be one of those builds.